src/core/audio/VorbisConsumer.cpp

```diff
@@ -27,7 +27,7 @@ auto VorbisConsumer::start(const std::string& filename) -> bool {
 
     this->consumerThread = std::thread([this, sfFile = std::move(sfFile), channels = channels] {
         auto lock{audioQueue.acquire_lock()};
-        auto buffer_size{static_cast<size_t>(std::max(0, 64 * channels))};
+        auto buffer_size{size_t(64 * channels)};
```

Comment:
```suggestion
        size_t buffer_size(64 * channels);
```

Reply:
This cast is actually required, since `64 * channels` is not size_t.